app_src/ui/screens/settings_screen.py

```
@@ -1248,6 +1300,25 @@ def update_label(self, seconds):
             self.ids.countdown_label.text = seconds
 
     def restart_service(self, *_):
+        try:
+            running = Service(name="Wallpapercarousel").is_running()
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)